### PR TITLE
[FIX] Closing sidebar when room menu is clicked.

### LINKED ISSUE #13841: Sidebar menu closes on clicking room menu (when screen-size is less.)
### Steps to reproduce:

1. Decrease the window size.
2. Click on the menu beside each room.
 

### Expected behavior:
Side bar should not close, the popover looks out of place now.

### Actual behavior:
side-bar

### Server Setup Information:

 Version of Rocket.Chat Server: 1.0.0-develop
 Operating System: Fedora-28
 Deployment Method: Meteor
 Number of Running Instances: 1
 DB Replicaset Oplog: Enables
 NodeJS Version: 8.11.4
 MongoDB Version: 3.6

PS: I don't think it is intended to close like that.

### PR DESCRIPTION
**Pull request by Kailash0311**
Closes #13841.
Stops propagation of "closing the menu event" onto the sidebar_menu class.

### CODE REVIEW

**Comment by Kailash0311** [comment]
I have added the test, can you please review it

File: tests/pageobjects/side-nav.page.js [hunk 1]
```diff
@@ -52,6 +52,8 @@ class SideNav extends Page {
 
 	get sidebarWrap() { return browser.element('.sidebar-wrap'); }
 
+	get sidebarItemMenu() { return browser.element('.sidebar-menu'); }
```
Comment (sampaiodiego) [inline tests/pageobjects/side-nav.page.js]: I haven't found this element on the source code. Maybe you were trying to get `sidebar-item__menu` instead?

Comment (Kailash0311) [reply tests/pageobjects/side-nav.page.js]: Yes sorry

**Comment by Kailash0311** [comment]
@sampaiodiego I've rectified the mistake.

**Pull request merged**